tower.yml

```diff
@@ -18,11 +18,11 @@ reports:
   "**/picard/*.MarkDuplicates.metrics.txt":
     display: "Picard: Metrics from CollectRnaMetrics"
   "**/picard/*_rna_metrics.txt":
-    display: "Picard: Metrics from MarkDuplicates"
-  "**/qualimap/qualimapReport.html":
-    display: "Qualimap HTML report from STAR_FOR_STARFUSION alignment"
-  "**/qualimap/rnaseq_qc_results.txt":
-    display: "Qualimap QC results from STAR_FOR_STARFUSION alignment in TXT format"
+    display: "GATK4: Metrics from MarkDuplicates"
+  "**/picard/*insert*size*metrics.txt":
+    display: "GATK4: Metrics from InsertSizeMetrics"
```

Comment:
If I understood what you meant earlier, I believe this is Picard for all except markduplicates